config/routes.rb

```diff
@@ -1,6 +1,19 @@
 Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
+  post '/auth/login', to: 'authentication#login'
+  resources :users do
+    resources :reservations
+  end
+
+  resources :stadiums do
+    resources :games
+  end
+
+  resources :teams, only: [:index] do
+  end
 end
```

Comment:
Only define routes that are needed (like with `:teams`)

Reply:
done